cmd/attack/kafka.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attack
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/fx"
+
+	"github.com/chaos-mesh/chaosd/cmd/server"
+	"github.com/chaos-mesh/chaosd/pkg/core"
+	"github.com/chaos-mesh/chaosd/pkg/server/chaosd"
+	"github.com/chaos-mesh/chaosd/pkg/utils"
+)
+
+func NewKafkaAttackCommand(uid *string) *cobra.Command {
+	options := core.NewKafkaCommand()
+	dep := fx.Options(
+		server.Module,
+		fx.Provide(func() *core.KafkaCommand {
+			options.UID = *uid
+			return options
+		}),
+	)
+
+	cmd := &cobra.Command{
+		Use:   "kafka <subcommand>",
+		Short: "Kafka attack related commands",
+	}
+
+	cmd.AddCommand(
+		NewKafkaFillCommand(dep, options),
+		NewKafkaFloodCommand(dep, options),
+		NewKafkaIOCommand(dep, options),
+	)
+
+	cmd.PersistentFlags().StringVarP(&options.Topic, "topic", "T", "", "the topic to attack")
+	_ = cmd.MarkPersistentFlagRequired("topic")
+
+	return cmd
+}
+
+func NewKafkaFillCommand(dep fx.Option, options *core.KafkaCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "fill [options]",
+		Short:             "Fill kafka cluster with messages",
+		ValidArgsFunction: cobra.NoFileCompletions,
+		Run: func(*cobra.Command, []string) {
+			options.Action = core.KafkaFillAction
+			utils.FxNewAppWithoutLog(dep, fx.Invoke(kafkaCommandFunc)).Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&options.Host, "host", "H", "localhost", "the host of kafka leader")
+	cmd.Flags().Uint16VarP(&options.Port, "port", "P", 9092, "the port of kafka leader")
```

Comment:
it seems that we can combine these two configs into one, like `address`

Reply:
Another question, is whether it must be set to the leader's address?

Reply:
> it seems that we can combine these two configs into one, like address
I prefer to not combine them, as many CLI tools like `mysql-client`, `ssh`, etc.

> Another question, is whether it must be set to the leader's address?
It seems unnecessary, I will change the description.

Reply:
The description is changed.